src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/FillsProcessor.kt

```diff
@@ -1,18 +1,42 @@
 package exchange.dydx.abacus.processor.wallet.account
 
+import exchange.dydx.abacus.output.SubaccountFill
 import exchange.dydx.abacus.processor.base.BaseProcessor
 import exchange.dydx.abacus.processor.base.mergeWithIds
+import exchange.dydx.abacus.protocols.LocalizerProtocol
 import exchange.dydx.abacus.protocols.ParserProtocol
+import indexer.codegen.IndexerFillResponseObject
 
-internal class FillsProcessor(parser: ParserProtocol) : BaseProcessor(parser) {
-    private val itemProcessor = FillProcessor(parser = parser)
+internal class FillsProcessor(
+    parser: ParserProtocol,
+    localizer: LocalizerProtocol?,
+    private val fillProcessor: FillProcessorProtocol = FillProcessor(parser = parser, localizer = localizer),
+) : BaseProcessor(parser) {
 
-    fun received(existing: List<Any>?, payload: List<Any>, subaccountNumber: Int): List<Any>? {
+    fun process(
+        existing: List<SubaccountFill>?,
+        payload: List<IndexerFillResponseObject>,
+        subaccountNumber: Int
+    ): List<SubaccountFill> {
+        val new = payload.mapNotNull { eachPayload ->
```

Comment:
`mapNotNull` not necessary here right? isn't `payload` a List of non-nullable IndexerFillResponseObject?

Reply:
It's necessary since item processor might return null (when required fields are missing).